sparkmagic/sparkmagic/tests/test_livysession.py

```diff
@@ -140,6 +140,7 @@ def test_constructor_starts_with_existing_session(self):
         assert session.id == session_id
         assert session.created_sql_context
         assert session._heartbeat_thread is None
+        assert u"heartbeatTimeoutInSecond" not in list(session.properties.keys())
```

Comment:
Someone could break this test unknowingly if `"heartbeatTimeoutInSecond": heartbeat_timeout` was removed from the `_create_session` method.